File: record_voice_android_app/app/src/main/java/com/amebaownd/pikohan_niwatori/recordsoundandtext/MainActivity.java
package com.amebaownd.pikohan_niwatori.recordsoundandtext

import androidx.appcompat.app.AppCompatActivity

import android.os.Bundle

import androidx.activity.viewModels

import com.amebaownd.pikohan_niwatori.recordsoundandtext.ui.home.MainViewModel
import com.amebaownd.pikohan_niwatori.recordsoundandtext.ui.home.getViewModelFactory

class MainActivity :AppCompatActivity(){

        val viewModel:MainViewModel by viewModels{getViewModelFactory()}

        override fun onCreate(savedInstanceState:Bundle?){
        super.onCreate(savedInstanceState)
        setContentView(R.layout.main_activity)

        }
        }

File: record_voice_android_app/app/src/main/java/com/amebaownd/pikohan_nwiatori/recordvoiceapp/MainActivity.kt
package com.amebaownd.pikohan_nwiatori.recordvoiceapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.amebaownd.pikohan_nwiatori.recordvoiceapp.ui.main.MainFragment

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.main_activity)
        if (savedInstanceState == null) {
            supportFragmentManager.beginTransaction()
                    .replace(R.id.container, MainFragment.newInstance())
                    .commitNow()
        }
    }
}
